### DISCUSSION TITLE
some errors like this: NVRM: krcWatchdog_IMPL: RC watchdog: GPU is probably locked!  Notify Timeout Seconds: 7

### DISCUSSION THREAD

**Discussion by zhangboin** [NVIDIA/open-gpu-kernel-modules#750] [Q&A]
Linux Kernel Version： 5.19.17
Hardware: NVIDIA GeForce GTX 1660 SUPER

When using the open source driver version: 560.35.03 the following error message is displayed during startup:

`Dec 13 13:37:28 localhost kernel: [    4.003343] NVRM: nvAssertFailedNoLog: Assertion failed: GPPut < WATCHDOG_GPFIFO_ENTRIES @ kernel_rc_watchdog.c:1417
Dec 13 13:37:29 localhost kernel: [    5.020905] NVRM: nvAssertFailedNoLog: Assertion failed: GPPut < WATCHDOG_GPFIFO_ENTRIES @ kernel_rc_watchdog.c:1417
Dec 13 13:37:30 localhost kernel: [    5.533498] igb 0000:05:00.0 eth0: igb: eth0 NIC Link is Up 1000 Mbps Full Duplex, Flow Control: RX/TX
Dec 13 13:37:30 localhost kernel: [    5.533845] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
Dec 13 13:37:37 localhost kernel: [   12.188909] NVRM: krcWatchdog_IMPL: RC watchdog: GPU is probably locked!  Notify Timeout Seconds: 7
Dec 13 13:37:38 localhost kernel: [   13.212942] NVRM: nvAssertFailedNoLog: Assertion failed: GPPut < WATCHDOG_GPFIFO_ENTRIES @ kernel_rc_watchdog.c:1417
Dec 13 13:37:43 localhost kernel: [   18.760898] NVRM: _threadNodeCheckTimeout: _threadNodeCheckTimeout: currentTime: 3d08be7d661300 >= 3d08be7d661300
Dec 13 13:37:43 localhost kernel: [   18.760901] NVRM: _threadNodeCheckTimeout: _threadNodeCheckTimeout: Timeout was set to: 4000 msecs!
Dec 13 13:37:43 localhost kernel: [   18.760905] NVRM: scrubberDestruct:  Timed out when waiting for the scrub to complete the pending work .`
Then Xorg failed to start。

What is the reason for the above error message? Or is there a solution?

**Comment by orrmaz** [comment]
Hi, I'm having the same issue. Did you found a solution for the issue?

**Reply by zhangboin** [reply]
Not yet. It appeared by accident when Xorg starting.

**Reply by Lensters** [reply]
I'm having the same issue.  Same card and driver, different kernel (6.8.0-60).  I'm running the latest MB bios.  I'm going to try moving to the 570 driver.

**Reply by Lensters** [reply]
Loading the 570 driver just made things worse.  I ended up reinstalling the OS.  It has been working solid for about a day now.
